model/patch/cli_intent.go

```diff
@@ -84,6 +87,7 @@ var (
 	cliProcessedAtKey   = bsonutil.MustHaveTag(cliIntent{}, "ProcessedAt")
 	cliIntentTypeKey    = bsonutil.MustHaveTag(cliIntent{}, "IntentType")
 	cliAliasKey         = bsonutil.MustHaveTag(cliIntent{}, "Alias")
+	cliCommitQueueKey   = bsonutil.MustHaveTag(cliIntent{}, "CommitQueue")
```

Comment:
I don't see that this is used anywhere - are you planning on saving this data into the patch document? I would think that we don't want this field to exist in the db since it's redundant data

Reply:
No longer necessary